src/scripts/components/audio/Audio.jsx

```diff
@@ -16,8 +17,11 @@ export default class Audio extends React.Component {
     this.state = {
       inEditMode: false,
       validTitle: true,
-      validContributors: true
+      validContributors: true,
+      playing: false,
+      pos: 0
     };
+    this.wavesurfer = Wavesurfer;
     this.audioId = props.match.params.id.split("-")[0];
     bindHandlers(this, [
```

Comment:
This suggestion isn't really necessary since you already wrote a service that handles it quite well, but I've been using this neat package called react-autobind (https://www.npmjs.com/package/react-autobind) that allows you to do lines 26 to 36 with an `autoBind(this)` in the constructor. It keeps your constructor super clean and gives you one less place to remember to update when you add a method to your React class.